clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -285,24 +310,24 @@ private EhcacheEntityResponse invokeLifeCycleOperation(ClientDescriptor clientDe
       case DESTROY_SERVER_STORE:
         destroyServerStore(clientDescriptor, (DestroyServerStore) message);
         break;
+      case CREATE_CLUSTERED_MAP:
+        createConcurrentMap(clientDescriptor, (CreateClusteredMap) message);
```

Comment:
Well, it is two messages _per client_ because the way it is implemented is based on ignoring result of `putIfAbsent` then doing `get`. And it creates added complexity it seems to me.